src/extensions/default/NavigationAndHistory/html/recentfiles-template.html

```diff
@@ -0,0 +1,15 @@
+<div id="mrof-container">
+    <div id="mrof-list-wrapper" class="quiet-scrollbars">
+        <div class="header">
+            <span>Recent Files</span>
+            <a class="close" id="mrof-list-close">×</a>
+        </div>
+        <span class="first pane-label"></span>
+        <span class="second pane-label"></span>
+        <ul id="mrof-list"></ul>
+        <div class="footer">
+            <span id="recent-file-path"></span>
+            <div class="btn-alt-quiet" id="clear-mrof-list" title="Clear recent files list">Clear</div>
```

Comment:
Make the strings "Recent Files", "Clear" and "Clear recent files list" translatable.